src/routes/auth.js

```diff
@@ -132,6 +133,29 @@ router.post(
 	})
 )
 
+router.get(
+	'/invites',
```

Comment:
@beblicarl upon further consideration and reading, we would rather want only actual users to use the `isAuthenticated` if we can help it

so instead of the `isAuthenticated` being used, the token will be part of the route instead

like `/invites/:token`, and then you actually decode the token in the code and return the appropriate error if it is invalid. Kindly do check out the `jsonwebtoken` library we are using to know how to decode

Apologies for the last-minute change. Kindly let me know if anything is not clear

Reply:
yes please